paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -128,11 +128,14 @@ internal class PaymentOptionsActivity : BaseSheetActivity<PaymentOptionResult>()
         viewBinding.continueButton.lockVisible = false
         viewBinding.continueButton.updateState(PrimaryButton.State.Ready)
 
-        viewBinding.continueButton.setDefaultBackGroundColor(
-            viewModel.config?.primaryButtonColor ?: ColorStateList.valueOf(
-                PaymentsThemeConfig.colors(baseContext.isSystemDarkTheme()).primary.toArgb()
+        viewModel.config?.let {
+            viewBinding.continueButton.setDefaultBackGroundColor(
+                it.primaryButtonColor ?: ColorStateList.valueOf(
+                    Color(it.appearance.getColors(baseContext.isSystemDarkTheme()).primary).toArgb()
```

Comment:
Will this work when the color is customized?

Reply:
Yup.

```                               
PaymentSheet.Configuration(                                       
  // ...                                      
 appearance = PaymentSheet.Appearance(                                          
   colorsLight = PaymentSheet.Colors.defaultLight.copy(
     primary = Color.Red.toArgb()                                            
    )                                        
  ),                                        
  primaryButtonColor = ColorStateList.valueOf(Color.Black.toArgb())                                    
)
```

gives me this payment sheet: 
![Screen Shot 2022-03-25 at 2 13 05 PM](https://user-images.githubusercontent.com/89166418/160201518-42fc1010-b4ee-4cc9-9002-c068b9cfbb1e.png)